tasks/Conformity Game.md

```diff
@@ -0,0 +1,70 @@
+
+Press any key to continue …
+
+2. The task was presented to participants as follows:
+
+Welcome to E-Mail Feud
+
+Now you have the chance to win some money!
+
+We are interested in how well members of an electronic group can predict what everyone else is thinking. In order to find out, we have invented a little game. We will ask you to predict what the majority of people in the group are thinking on a topic. You make a prediction, and on the next session you will find out what others in the group thought. Then you can revise your choice. If, at the end, you are right, you can win an extra bonus of $1.00 for each question in which you succeed in predicting the right answer.
+
+Ready? Here we go …
+
+3. Participants were then prompted to choose between two alternatives as their initial position:
+
+Now, see if you can predict the mathematician (Euler or Hilbert) which the majority of your groupmates will choose. For an additional $1.00, on the final session the most popular mathematician will be?___(Choose E or H)
+How strongly do you feel about this (1 = not strongly to 7 = very strongly)?
+
+4. Participants entered their initial prediction (in many cases it was a blind guess) by pressing either the letter E or the letter H and a number (between 1 and 7) indicating their degree of conviction. The word “SENDING …” then appeared briefly on the screen, followed by “SENT,” and one round of communication on that topic was completed. The next round of communication took place at the next session, 2 or 5 days later. On subsequent days, participants were reminded of their choice in the previous session and were shown the choices their neighbors had made. They were then allowed to make a new choice.
+
+Last time you said “Hilbert ”
+Person A said “Euler ” with conviction 6
+Person C said “Euler ” with conviction 7
+Person F said “Hilbert ” with conviction 7
+Person W said “Euler ” with conviction 6
+
+# Criteria
+## Performance calculation
+Percent change and confidence level, from 1 to 7, were independent variables and number of opposing messages was a dependent variable. Although expressed conviction declined continuously with increased numbers of people providing an opposing message, change was much less linear. As expected, very few people (less than 3% overall) changed when 4, 3, or even 2 others agreed with their position. However, when they received three or more opposing messages, substantially higher percentages (59%) of individuals changed their choice. Although individuals were surprisingly stubborn, weighting their own opinions highly even when faced with unanimous opposition, they were generally quite responsive to social influence.
+
+## Incentives
+Each participant recieved a base payment, and then was paid $1 for predicting each answer of the majority.
```

Comment:
Change to "Each participant received a base payment regardless of performance. Participants could then earn $1.00 each time they correctly predicted the majority answer.